packages/css/src/components/image/image.scss

```diff
@@ -3,12 +3,20 @@
  * Copyright Gemeente Amsterdam
  */
 
+@import "../aspect-ratio/mixins";
+
 .ams-image {
   block-size: auto; /* [1] */
   font-style: italic; /* [3] */
   max-inline-size: 100%; /* [1] */
   vertical-align: middle; /* [2] */
 
+  &[class*="ams-image-aspect-ratio--"] {
+    @include ams-aspect-ratio;
+  }
+
+  @include generate-aspect-ratio-classes("-aspect-ratio");
+
   &--cover {
     object-fit: cover;
```

Comment:
Yeah, and I feel that adding `object-fit: contain` would be a good use-case as well. Why wasn't that added?

Reply:
That will render the image smaller than its Grid Cell, which makes the alignment look off. Or do you see other use cases where this would be useful?